index.js

```diff
@@ -162,10 +137,19 @@ router.start = router.trigger = function () {
 
 };
 
-router.redirect = function (route) {
+router.redirect = function (url, replace) {
```

Comment:
blind try to make redirect replace state by default.
needs https://github.com/christianalfoni/addressbar/issues/4 live and also some tests